engine/processReports/src/main/scala/pl/touk/nussknacker/processCounts/influxdb/ProcessBaseCounts.scala

```diff
@@ -0,0 +1,23 @@
+package pl.touk.nussknacker.processCounts.influxdb
+
+case class ProcessBaseCounts(nodes: Map[String, Long]) {
+
+  //node ids in influx are different than original ones, i.e influx converts spaces and dots to dashes '-'
```

Comment:
InfluxDB allows for dots in names, it must be our code (or some library we use). InfluxDB itself happily accepts and returns dots (just tested in 1.8.3).

Quick test:

Session 1: `docker run --rm -itP --name influx influxdb:1.8.3`
Session 2: `docker exec -it influx influx -precision rfc3339`
```
> create database test;
> use test
Using database test
> insert test,a=a.b f=1 0
> select * from test
name: test
time                 a   f
----                 -   -
1970-01-01T00:00:00Z a.b 1
```

Session 3: `docker exec -it influx curl -XPOST -d 'q=select * from test' 'localhost:8086/query?db=test'`
```json
{"results":[{"statement_id":0,"series":[{"name":"test","columns":["time","a","f"],"values":[["1970-01-01T00:00:00Z","a.b",1]]}]}]}
```

Reply:
It think this was the case when we used to send metrics via graphite protocol.... I added comment to remove it when we remove useLegacyMetrics flag

Reply:
Oh, that explains it. I totally forgot about that abomination (despite the fact that we still use it)...